ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -89,6 +89,21 @@
   @Default("false")
   public final boolean storeEnableHardDelete;
 
+
+  /**
+   * How many disks should be started in parallel.
+   */
+  @Config("store.num.disk.startup.threads")
+  @Default("10")
```

Comment:
I decided to just spawn a thread for each store during startup, so I took out both of these configurations.